lib/truffle/thread.rb

```diff
@@ -29,4 +29,16 @@ class ThreadError < StandardError
 class Thread
   Queue = ::Queue
   SizedQueue = ::SizedQueue
+
+  class Queue
+    def pop(non_block = false, timeout: nil)
+      Primitive.queue_pop(
+        self,
+        Primitive.as_boolean(non_block),
+        Truffle::QueueOperations.validate_and_prepare_timeout(non_block, timeout),
```

Comment:
Trailing comma for method arguments is AFAIK never used so far, I think it'd make sense to use an existing RuboCop rule to keep that consistent.
Could you remove the trailing comma and move the `)` at the end of this line?